From: Eric Macdonald <eric.macdonald@windriver.com>
Date: Fri, 14 Aug 2026 13:10:42 -0400
Subject: Disable binds verification for ostree pull

This patch adds the --disable-verify-bindings option to both ostree
pull commands.

Signed-off-by: Eric Macdonald <eric.macdonald@windriver.com>
---
 init-ostree-install.sh | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/init-ostree-install.sh b/init-ostree-install.sh
index 315c466..83a7509 100644
--- a/init-ostree-install.sh
+++ b/init-ostree-install.sh
@@ -1523,7 +1523,7 @@ if [ "$INSTL" != "" ] ; then
 	fi
 fi
 
-cmd="ostree pull $lpull --repo=${PHYS_SYSROOT}/ostree/repo ${INSTNAME} ${INSTBR}"
+cmd="ostree pull $lpull --disable-verify-bindings --repo=${PHYS_SYSROOT}/ostree/repo ${INSTNAME} ${INSTBR}"
 echo running: $cmd
 $cmd || fatal "Error: ostree pull failed"
 [ -d /${instl_name} ] && rm -rf /${instl_name}
@@ -1570,7 +1570,7 @@ if [ "$INSTAB" = "1" ] ; then
 	if [ ${INSTURL#http} = ${INSTURL} ]; then
 		localcache="--localcache-repo=${PHYS_SYSROOT}/ostree/repo"
 	fi
-	ostree pull $lpull --repo=${PHYS_SYSROOT}_b/ostree/repo ${localcache} ${INSTNAME}:${INSTBR} || fatal "ostree pull failed"
+	ostree pull $lpull --disable-verify-bindings --repo=${PHYS_SYSROOT}_b/ostree/repo ${localcache} ${INSTNAME}:${INSTBR} || fatal "ostree pull failed"
 	ostree admin deploy ${kargs_list} --sysroot=${PHYS_SYSROOT}_b --os=${INSTOS} ${INSTBR} || fatal "ostree deploy failed"
 fi
 
